From: SeongJae Park <sj@kernel.org>
Date: Mon, 20 Jan 2025 13:43:55 -0800
Subject: [PATCH] ==== DAMOS filter type unmapped ====

mm/damon: introduce DAMOS filter type UNMAPPED

User decides whether their memory will be mapped or unmapped.  It
implies that the two types of memory can have different characteristics
and maangement requirements.  Provide the DAMON-observaibility
DAMOS-operation capability for the different types by introducing a new
DAMOS filter type for unmapped pages.

Changes from RFC
(https://lore.kernel.org/20241127205624.86986-1-sj@kernel.org)
- Rebase on latest mm-unstable
- Wordsmith commit message
- Add documentation

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_3t3rr6kc | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_3t3rr6kc

diff --git a/hkml_cv_bogus/hkml_cv_bogus_3t3rr6kc b/hkml_cv_bogus/hkml_cv_bogus_3t3rr6kc
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

